e2e/src/usecases/FiatConnectTransferOut.js

```diff
@@ -74,62 +77,210 @@ async function submitTransfer() {
+    await enterAccountInformation()
 
     await submitTransfer()
 
     // ******** Returning user experience ************
-    await navigateToFiatExchangeScreen()
-    await waitForElementId('cashOut')
-    await element(by.id('cashOut')).tap()
+    await returnUserTransferOut(token, cashOutAmount)
+  })
+}
 
-    await selectCurrencyAndAmount(token, cashOutAmount)
+export const fiatConnectKycTransferOut = () => {
+  it('FiatConnect cash out', async () => {
+    const platform = device.getPlatform()
+    if (platform == 'android') {
+      return
+    }
 
-    await submitTransfer()
+    // ******** First time experience ************
+    const cashOutAmount = 0.01
+    const gasAmount = 0.005
+    const fundingAmount = `${2 * cashOutAmount + gasAmount}`
+    const token = 'cUSD'
+    const walletAddress = await onboardAndBeginTransferOut(token, fundingAmount, cashOutAmount)
+
+    // KycLanding
+
+    // Step 1
+    await expect(element(by.text('Verify your Identity'))).toBeVisible()
+    await expect(element(by.id('step-one-grey'))).not.toBeVisible()
+    await expect(element(by.id('step-two-grey'))).toBeVisible()
+    await expect(element(by.id('checkbox/unchecked'))).toBeVisible()
+    await element(by.id('checkbox/unchecked')).tap()
+    await element(by.id('PersonaButton')).tap()
+
+    // Persona
+    await waitFor(element(by.text('Begin verifying')))
+      .toBeVisible()
+      .withTimeout(5000)
+    await element(by.text('Begin verifying')).tap()
+
+    // Country of govt id screen
+    await expect(element(by.text('Select'))).toBeVisible()
+    await element(by.text('Select')).tap()
+
+    // Id select screen
+    await expect(element(by.text('Driver License'))).toBeVisible()
+    await element(by.text('Driver License')).tap()
+    await expect(element(by.text('Enable camera'))).toBeVisible()
+    await element(by.text('Enable camera')).tap()
+
+    // Manually wait for Take Photo button to appear, withTimeout didn't work
+    await sleep(10000)
+
+    // License photo
+    await element(by.label('Take photo')).tap()
+    await element(by.text('Use this photo')).tap()
+
+    // Selfie section
+    await waitFor(element(by.text('Get started')))
+      .toBeVisible()
+      .withTimeout(10000)
+    await element(by.text('Get started')).tap()
+    await element(by.label('shutter button')).tap()
+    await element(by.label('shutter button')).tap()
+    await element(by.label('shutter button')).tap()
+
+    // Name, number, address form
+    await waitFor(element(by.text('Phone Number')))
+      .toBeVisible()
+      .withTimeout(10000)
+    await element(by.type('PersonaPhoneNumberKit2.PhoneNumberTextField')).typeText('0123456789')
+    await element(by.text('Phone Number')).tap() // Tap away to unfocus from input
+    await element(by.text('Continue')).tap()
+
+    await element(by.text('Done')).tap() // End of Persona flow
+
+    // Check that Mock Provider info is defined
+    if (MOCK_PROVIDER_BASE_URL && MOCK_PROVIDER_API_KEY) {
```

Comment:
should the else part be an error?

Reply:
That part is for developers who are external to Valora who also want to run the e2e tests. I don't think they should fail because they don't have access to the encrypted env variables, which is akin to what is [currently done](https://github.com/valora-inc/wallet/blob/main/e2e/src/usecases/NewAccountPhoneVerification.js#L54) for the `NewAccountPhoneVerification` test.

Reply:
could the entire test be skipped instead of partially running the test if this is the case?

Reply:
Yes, just updated the test.